feature-libs/my-account/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@spartacus/myaccount",
```

Comment:
What about `@spartacus/my-account` name? 
We need to specify naming convention for the libs

Reply:
Agreed. Added convention section to the readme and changed to `my-account`.